recipes/pyopcode/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+BUILD_CONFIG=Release
+
+if [ `uname` == Linux ]; then
+    CC=${PREFIX}/bin/gcc
+    CXX=${PREFIX}/bin/g++
+
+    # FIXME refactor to reuse the python name (e.g. python3.5m)
+    # FIXME detect any kind of suffix (m, or d)
+    include_path=${PREFIX}/include/python${PY_VER}
+    if [ ! -d $include_path ]; then
+      # Control will enter here if $DIRECTORY doesn't exist.
+      include_path=${PREFIX}/include/python${PY_VER}m
+    fi
+
+    PY_LIB="libpython${PY_VER}.so"
+    PYTHON_LIBRARY=${PREFIX}/lib/${PY_LIB}
+    if [ ! -f $PYTHON_LIBRARY ]; then
+        PYTHON_LIBRARY=${PREFIX}/lib/libpython${PY_VER}m.so
+    fi
+
+    cd pyopcode
+    mkdir build
+    cd build
+
+    cmake ../src \
+        -Wno-dev \
```

Comment:
Ack, sorry, I meant to put that comment in the `bld.bat`. Have now added it up there.